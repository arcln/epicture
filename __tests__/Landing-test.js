import 'react-native';
import React from 'react';
import Landing from '../screens/LandingScreen';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';

describe('Landing snapshot', () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it('renders the screen', async () => {
    const tree = renderer.create(<Landing />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
