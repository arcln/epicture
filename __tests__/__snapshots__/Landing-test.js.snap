// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing snapshot renders the screen 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "opacity": 0,
      }
    }
  >
    <ExponentLinearGradient
      colors={
        Array [
          4281216642,
          4279702852,
        ]
      }
      style={
        Object {
          "height": 1334,
          "position": "absolute",
          "width": 750,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "space-around",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 2,
          "flexDirection": "column",
          "justifyContent": "space-around",
          "opacity": 0,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "height": 300,
            "position": "absolute",
            "top": 599,
            "transform": Array [
              Object {
                "translateY": 0,
              },
            ],
          }
        }
      >
        <Image
          onError={[Function]}
          onHeightChange={[Function]}
          source={1}
          style={
            Array [
              Object {
                "height": 0,
                "width": 750,
              },
              undefined,
            ]
          }
        />
      </View>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "justifyContent": "center",
          "opacity": 0,
          "paddingBottom": 333.5,
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityStates={Array []}
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {},
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#007AFF",
                  "fontSize": 18,
                  "padding": 8,
                  "textAlign": "center",
                },
                Object {
                  "color": "#1bb76e",
                },
              ]
            }
          >
            Login
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
        "opacity": 0,
        "position": "absolute",
        "top": 637,
        "width": 750,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#fff",
        }
      }
    >
      The magic of the internet
    </Text>
  </View>
  <View
    style={
      Object {
        "opacity": 1,
        "position": "absolute",
        "top": 599,
        "transform": Array [
          Object {
            "translateY": 0,
          },
        ],
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "flexDirection": "column",
          "justifyContent": "space-around",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "height": 300,
          }
        }
      >
        <Image
          onError={[Function]}
          onHeightChange={[Function]}
          source={1}
          style={
            Array [
              Object {
                "height": 0,
                "width": 750,
              },
              undefined,
            ]
          }
        />
      </View>
    </View>
  </View>
</View>
`;
